ow_lander/scripts/unstow_client.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+from ow_lander.actions import unstow
+from ow_lander import node_helper
+
+import argparse
+
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
```

Comment:
There's new code like this with 4-space indentation. It would be good to make it 2-space to be consistent with the bulk of the code in this PR.

Reply:
Done.